care/facility/api/viewsets/patient.py

```diff
@@ -2,11 +2,12 @@
 import json
 from json import JSONDecodeError
 from config.celery_app import app
-
+from django.conf import settings
```

Comment:
Codacy found an issue: [third party import "from django.conf import settings" should be placed before "from config.celery_app import app"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)